aten/src/ATen/CMakeLists.txt

```diff
@@ -79,6 +79,9 @@ file(GLOB miopen_cpp "miopen/*.cpp")
 file(GLOB mkl_cpp "mkl/*.cpp")
 file(GLOB mkldnn_cpp "mkldnn/*.cpp")
 
+file(GLOB nestedtensor_h "native/nestedtensor/*.h")
+file(GLOB nestedtensor_cpp "native/nestedtensor/*.cpp")
```

Comment:
I hope you're prepared to update all the bzl build files lol, we definitely do not recursively glob under native 📦